pkg/globalmanager/config/config.go

```diff
@@ -58,6 +59,10 @@ type ControllerConfig struct {
 
 	// kb config to info the worker
 	KB KBConfig `json:"knowledgeBaseServer,omitempty"`
+
+	// period config min resync period
+	// default 30s
+	MinResyncPeriod int64 `json:"minResyncPeriod,omitempty"`
```

Comment:
Better to specify the second unit, see https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#units

```suggestion
	MinResyncPeriodSeconds int64 `json:"minResyncPeriodSeconds,omitempty"`
```